tsutils/user_interaction.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+import re
+
+
+async def doubleup(ctx, message):
+    """Edit the last message to x2 or more if it's being repeated"""
```

Comment:
>      """Edit the last message to x2 or more if it's being repeated""" 

can you say, `     """Edit the last message to include the string `x2` or more if would otherwise be repeated""" `

I think this is more clear